test/e2e/scenario/logs.scenario.ts

```diff
@@ -75,6 +75,39 @@ describe('logs', () => {
       })
     })
 
+  createTest('read only the first bytes of the response')
+    .withLogs({ forwardErrorsToLogs: true })
+    .run(async ({ serverEvents, baseUrl, servers }) => {
+      await browserExecuteAsync((done) => {
+        fetch('/throw-large-response').then(
+          (response) => {
+            // The body stream needs to be cancelled, else the browser will still download the whole
+            // response even if it is unused.
+            response
+              .body!.getReader()
+              .cancel()
+              .catch((error) => console.log(error))
+            done(undefined)
+          },
+          (error) => console.log(error)
+        )
+      })
+
+      await flushEvents()
+      expect(serverEvents.logs.length).toBe(1)
+      expect(serverEvents.logs[0].message).toBe(`Fetch error GET ${baseUrl}/throw-large-response`)
+      expect(serverEvents.logs[0].error?.origin).toBe('network')
+      expect(serverEvents.logs[0].error?.stack?.length).toBeLessThan(LARGE_RESPONSE_MIN_BYTE_SIZE)
+      expect(servers.base.app.getLargeResponseWroteSize()).toBeLessThan(LARGE_RESPONSE_MIN_BYTE_SIZE)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it could be nice to have an assertion ensuring that we have indeed read something from the response, `stack.length` and `wroteSize` could be `0`.
Maybe ensuring that those values are close to our `32kb` limit.